### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by Roman-Scherbakov** [avocode/avocode#525]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.3-3630f5a
Source UID: 87504
```
Error: invalid literal/lengths set
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by rychlis** [comment]
Thanks for your report. Does this error happen regularly or rather rarely? It happens when the document is not downloaded properly but we definitely have to handle this error better.

It's a duplicate of #447 so I am going to keep just that one bugreport open. Also consider updating to 2.4.5 which fixed a lot of other bugs: https://avocode.com/download/

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
